ci-operator/step-registry/assisted/baremetal/operator/hypershift/assisted-baremetal-operator-hypershift-ref.yaml

```diff
@@ -0,0 +1,24 @@
+ref:
+  as: assisted-baremetal-operator-hypershift
+  from: assisted-service-scripts
+  commands: assisted-baremetal-operator-hypershift-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  timeout: 1h30m0s
```

Comment:
It would invoke deploy_spoke_cluster.sh, so unfortuanely could take a while. Maybe will have to even increase it:/

Reply:
I thought it should be lighter than the regular flow
Anyway, let's see how it goes

Reply:
is this timeout value the expected high-bound for this process, or can it be lowered?